DSharpPlus.UnifiedCommands/Message/MessageOptionAttribute.cs

```diff
@@ -0,0 +1,22 @@
+namespace DSharpPlus.UnifiedCommands.Message;
+
+/// <summary>
+/// The option attribute. Decides what name a option has.
+/// </summary>
+[AttributeUsage(AttributeTargets.Parameter)]
+public class MessageOptionAttribute : Attribute
+{
+    public string Option { get; set; }
+    public string? ShorthandOption { get; set; }
```

Comment:
there might be merit in turning this into a `char`

Reply:
Some shorthands might be more than one character. But usually that isn't the case. But I think we should keep shorthand option as a string for those people.